test/tests/custom typescript path/eslintrc.js

```diff
@@ -9,52 +9,10 @@ module.exports = {
         "sourceType": "module"
     },
     "plugins": [
-        "@typescript-eslint",
-        "@typescript-eslint/tslint"
+        "@typescript-eslint"
     ],
     "rules": {
         "@typescript-eslint/array-type": "error",
-        "@typescript-eslint/interface-name-prefix": "error",
-        "@typescript-eslint/member-ordering": "off",
-        "@typescript-eslint/no-empty-function": "off",
-        "@typescript-eslint/no-explicit-any": "off",
-        "@typescript-eslint/no-param-reassign": "off",
-        "@typescript-eslint/no-parameter-properties": "off",
-        "@typescript-eslint/no-use-before-declare": "off",
-        "@typescript-eslint/promise-function-async": "off",
-        "@typescript-eslint/unbound-method": "off",
-        "arrow-body-style": "off",
-        "default-case": "off",
-        "linebreak-style": "off",
-        "no-bitwise": "off",
-        "no-empty": "off",
-        "no-magic-numbers": "off",
-        "padding-line-between-statements": [
-            "off",
-            "error",
-            {
-                "blankLine": "always",
-                "prev": "*",
-                "next": "return"
-            }
-        ],
-        "prefer-template": "off",
-        "unicorn/filename-case": "off",
-        "@typescript-eslint/tslint/config": [
```

Comment:
To add in this PR: there should be a test that exercises the `tslint/config` rule.